### PR TITLE
[CR-1526] Document checkout sdk display option

### PR DESCRIPTION
**Pull request by antonioliv**

### CODE REVIEW

File: content/en/checkout/reference.md [hunk 1]
```diff
@@ -38,58 +39,58 @@ An object with the following properties:
     - `splits`: optional `Array` of `Object`s used in split payments. Each `Object` has the following properties:
       - `split_key`: `string` (<= 50 characters) with the merchant's internal key for identifying the split.
       - `split_descriptive`: `string` (<= 255 characters) with the description of the split.
-      - `value` (*required*): `number` the split funds, rounded to 2 decimals.
+      - `value` (_required_): `number` the split funds, rounded to 2 decimals.
```
Comment (Arlanthir) [inline content/en/checkout/reference.md:42]: Why these changes?

**Comment by Espina2** [comment]
You have to give a chill pill to your linter @antonioliv :D

File: content/en/checkout/reference.md (hunk 1 above)
Comment (antonioliv) [reply content/en/checkout/reference.md:42]: It was the linter probably. It's the same output just didn't know that the linter was so aggressive on that 🤔

**Comment by antonioliv** [comment]
Indeed 😓

File: content/en/checkout/guide.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Guide
-description: ""
+description: ''
```
Comment (Arlanthir) [inline content/en/checkout/guide.md:3]: Can you revert these linter changes so we keep it consistent with the other files?

File: content/en/checkout/reference.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Reference
-description: ""
+description: ''
```
Comment (Arlanthir) [inline content/en/checkout/reference.md:3]: String change as well.

File: content/en/checkout/reference.md [hunk 4]
```diff
@@ -19,8 +19,9 @@ Creates a Checkout Session.
 <!-- TODO: Better way to show this! -->
 
 An object with the following properties:
-- `type` (*required*): `Array` of `string`s representing which [payment type(s)](/concepts/payment-types) this Checkout session should accept. For now, only `['single']` is supported.
-- `payment` (*required*): `Object` describing the payment configuration, with the following properties:
+
+- `type` (_required_): `Array` of `string`s representing which [payment type(s)](/concepts/payment-types) this Checkout session should accept. For now, only `['single']` is supported.
```
Comment (Arlanthir) [inline content/en/checkout/reference.md:23]: Can you revert these linter changes so we keep it consistent with the other files (the italics)?

**Review by Arlanthir — APPROVED** [review]
LGTM
